server/routes/teacherPriviledge.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import UserController from '../controllers/userController';
+import ResultsController from '../controllers/resultsController';
+import AddResultsValidation from '../middlewares/addResultsValidation';
+import { checkIfUserHasSchool } 
+    from '../middlewares/checkUser';
+
+const router = express.Router();
+
+
+router.use(checkIfUserHasSchool);
+
+// get students from the database
+router.get('/api/v1/users/:query', UserController.getUsers);
```

Comment:
A question about structuring the API routes:

Do you think this should be `/api/v1/users/getusers/:query` or we should leave it as `/api/v1/users/:query`? 

We will be adding a `getUser` route for the individual user, so I was wondering whether that would then be `/api/v1/users/getuser` or something along those lines

Reply:
@jamsgra Currently, the endpoint is /api/v1/users. We can still have /api/v1/user to get user profile(info).